bin/accessibility-automation/cypress/index.js

```diff
@@ -424,7 +424,11 @@ Cypress.Commands.add('getAccessibilityResults', () => {
 	}
 });
 
-Cypress.Commands.addQuery('performScanSubjectQuery', function (chaining, setTimeout) {
-    this.set('timeout', setTimeout);
-    return () => cy.getSubjectFromChain(chaining);
-});
+if (!Cypress.Commands.hasOwnProperty('_myQueryAdded')) {
```

Comment:
Can we make this a bit more context specific, maybe something like
```suggestion
if (!Cypress.Commands.hasOwnProperty('_browserstackSDKQueryAdded')) {
```

Reply:
Done.